fastlane_bot/events/managers/base.py

```diff
@@ -172,18 +172,31 @@ def set_carbon_v1_fee_pairs(self):
         """
         Set the carbon v1 fee pairs.
         """
-        if "carbon_v1" in self.exchanges:
```

Comment:
I think that by changing this `if` statement into:
```
for ex in [ex for ex in self.cfg.CARBON_V1_FORKS if ex in self.exchanges]:
```
You could keep most of the code below exactly as is.
Fewer changes == less risk AND better review.

Reply:
I have implemented your suggestion. thanks